services/web/server/src/simcore_service_webserver/computation_api.py

```diff
@@ -346,13 +352,6 @@ async def _update_task(
             for task_row in tasks_rows:
                 # for some reason the outputs are not present in the
                 # tasks outputs. copy them over (will be used below)
-
```

Comment:
this piece has been on/off for so long ...

Reply:
I will have a look at it again when I move this stuff to the director..